pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdClusters.java

```diff
@@ -268,11 +268,9 @@ void run() throws Exception {
             try {
                 processArguments();
             } catch (Exception e) {
-                System.err.println(e.getMessage());
-                System.err.println();
                 String chosenCommand = jcommander.getParsedCommand();
                 getUsageFormatter().usage(chosenCommand);
-                return;
+                throw e;
```

Comment:
What does the output of a failed call look like now?

Reply:
Previously:

```
$ bin/pulsar-admin sources create --source-config-file doesnotexist.yaml
doesnotexist.yaml (No such file or directory)

Submit a Pulsar IO source connector to run in a Pulsar cluster
Usage: create [options]
... other usage prints, and exit with 0
```

Now:

```
$ bin/pulsar-admin sources create --source-config-file doesnotexist.yaml

Submit a Pulsar IO source connector to run in a Pulsar cluster
Usage: create [options]
  Options:
    -a, --archive
... other usage prints

doesnotexist.yaml (No such file or directory)
# exit with 1
```

Reply:
nice, thanks for sharing!